.gitmodules

```diff
@@ -0,0 +1,3 @@
+[submodule "libraries/standard/http/third_party/http_parser"]
```

Comment:
As this library is specific to HTTP, it is fine to keep it under the `http` folder for now. However, for shared 3rd party libraries like `tinycbor` or a JSON utility, there would be a common folder created. Would this library be moved there then?

Reply:
I guess putting `http_parser` under `http` makes it clear that `http` is the only library using it. But I am already using a common folder for `CMock` because it will likely be used by all libraries.